### PR TITLE
Update layout components to MUI

### PR DESCRIPTION
**Pull request by pbadillatorrealba**
In this pr the general layout of the application (navbar and page container) is updated by the Material UI components.

Note: Depends of pr #34

### CODE REVIEW

File: DashAI/front/src/components/ResponsiveAppBar.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import MenuItem from "@mui/material/MenuItem";
+import { Link as RouterLink, useLocation } from "react-router-dom";
+import { useTheme } from "@mui/material/styles";
+import HomeIcon from "@mui/icons-material/HomeOutlined";
+
+const pages = [
+  { name: "Data", to: "/data", disabled: false },
+  { name: "Experiments", to: "/experiments", disabled: false },
+  { name: "Results", to: "/results", disabled: false },
+  { name: "Play", to: "/play", disabled: true },
```
Comment (cristian-tamblay) [inline DashAI/front/src/components/ResponsiveAppBar.jsx]: For the serving from the backend works, path starts with /app/, i.e
```suggestion
  { name: "Data", to: "/app/data", disabled: false },
  { name: "Experiments", to: "/app/experiments", disabled: false },
  { name: "Results", to: "/app/results", disabled: false },
  { name: "Play", to: "/app/play", disabled: true },
```
If not they will be treated as static files from the backend

**Review by cristian-tamblay — CHANGES_REQUESTED** [review]

File: DashAI/front/src/App.jsx [hunk 2]
```diff
@@ -23,27 +26,45 @@ function App() {
...
+              direction="row"
+              justifyContent="center"
+              alignItems="center"
+            >
+              <Routes>
+                <Route path="/" element={<Home />} />
+                <Route path="/data/" element={<Data />} />
+                <Route path="/experiments" element={<Experiment />} />
+                <Route path="/results" element={<Results />} />
+                <Route path="/play" element={<Play />} />
```
Comment (cristian-tamblay) [inline DashAI/front/src/App.jsx]: Same as other comment
```suggestion
                <Route path="/app/" element={<Home />} />
                <Route path="/app/data" element={<Data />} />
                <Route path="/app/experiments" element={<Experiment />} />
                <Route path="/app/results" element={<Results />} />
                <Route path="/app/play" element={<Play />} />
```

File: DashAI/front/src/components/ResponsiveAppBar.jsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+              ))}
+            </Menu>
+          </Box>
+
+          {/* Render on sm to xl screens */}
+          <Box sx={{ flexGrow: 1, display: { xs: "none", sm: "flex" } }}>
+            <IconButton
+              aria-label="delete"
+              component={RouterLink}
+              to="/"
```
Comment (cristian-tamblay) [inline DashAI/front/src/components/ResponsiveAppBar.jsx]:
```suggestion
              to="/app"
```

File: DashAI/front/src/components/ResponsiveAppBar.jsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+            src="/images/logo.png"
+            variant="square"
+            sx={{ width: 120, p: 0, mr: 3, my: 1, mt: 2 }}
+          />
+
+          {/* Render on xs screens */}
+          <Box sx={{ flexGrow: 1, display: { xs: "flex", sm: "none" } }}>
+            <IconButton
+              size="large"
+              aria-label="account of current user"
```
Comment (cristian-tamblay) [inline DashAI/front/src/components/ResponsiveAppBar.jsx]: I guess this comes from a template, we don't have user accounts

File: DashAI/front/src/components/ResponsiveAppBar.jsx (hunk 1 above)
Comment (pbadillatorrealba) [reply DashAI/front/src/components/ResponsiveAppBar.jsx]: I forgot about it. Changed!

File: DashAI/front/src/App.jsx (hunk 2 above)
Comment (pbadillatorrealba) [reply DashAI/front/src/App.jsx]: Changed

File: DashAI/front/src/components/ResponsiveAppBar.jsx (hunk 4 above)
Comment (pbadillatorrealba) [reply DashAI/front/src/components/ResponsiveAppBar.jsx]: You guessed right. Deleted

File: DashAI/front/src/components/ResponsiveAppBar.jsx (hunk 3 above)
Comment (pbadillatorrealba) [reply DashAI/front/src/components/ResponsiveAppBar.jsx]: Changed

**Review by cristian-tamblay — APPROVED** [review]

**Pull request merged**